tests/unit/state/test_queue_adapter.py

```diff
@@ -57,7 +57,7 @@ def tearDownClass(cls) -> None:
 
     def test_construction(self):
         self.assertEqual(
-            2,
+            3,
```

Comment:
@pmrv Do you understand why the number of matches increased? I just want to prevent that the same files are read twice.

Reply:
The new code finds these three files
```python
['/home/ponder/science/phd/dev/pyiron_base/testqueueadapters/res1/queues/queue.yaml',
 '/home/ponder/science/phd/dev/pyiron_base/testqueueadapters/res2/queues/queue.yaml', 
'/home/ponder/science/phd/dev/pyiron_base/testqueueadapters/res2/queues/clusters.yaml']
```
which are the ones that are created in the `setUpClass`.  It then creates a queue adapter for each of the files. But the way QueueAdapter works is that it wants to be passed the directory that the queue/cluster configuration, so the last two files give the same queue adapter.  It's an easy fix, but going forward do we even want to search for `clusters.yaml` anymore?

Reply:
Yes, the `clusters.yaml` is used in the case when pyiron connects to multiple HPC clusters with different queuing systems. But I think it is fine, with the recent changes it only looks for directories and then `pysqa` is handling the rest.